scripts/aws/monitor.py

```diff
@@ -221,30 +248,32 @@ def add_cloudwatchlog_role_to_apigateway(client, role_arn):
     # Sleep for 10 seconds to allow the role created above to be avialable
     time.sleep(10)
     api_client = session.create_client('apigateway', args.aws_region)
+    enable_cloudwatchlogs_in_stages(api_client, args.gw_id, args.logging_level)
     add_cloudwatchlog_role_to_apigateway(api_client, cloudwatch_log_role_arn)
 
+    api_gateway_name = get_api_gateway_name(api_client, args.gw_id)
     sns_client = session.create_client('sns', args.aws_region)
     cw = session.create_client('cloudwatch', args.aws_region)
 
     create_api_alarm(cw, 'api_count', 'Count', 'ApiGateway',
                      'Average', 'GreaterThanOrEqualToThreshold',
                      'Average API count for a period of 5 min', 50, 300, 1,
-                     [{'Name': 'ApiName', 'Value': args.api_gw}, {'Name': 'Label', 'Value': args.api_stage}],
+                     [{'Name': 'ApiName', 'Value': api_gateway_name}, {'Name': 'Label', 'Value': args.api_stage}],
```

Comment:
Same question here - why use a potentially redundant signifier like name? You could have `ID: ApiId` and make this alarm far less likely to be confusing. You should consider adding it as a new field or replacing the name field.